code/game/sound.dm

```diff
@@ -342,7 +344,7 @@ var/const/FALLOFF_SOUNDS = 0.5
 
 		else if (pressure_factor < 0.5)
 			S.environment = SPACE
-		else
+		else if(!istype(src, /mob/new_player/))//again we may be in lobby
```

Comment:
so this is duct tape again, code below seems to be copy-pasted, as it exists in "for humans" part, but i'm not sure.